storage/integration_test.go

```diff
@@ -252,103 +252,161 @@ func config(ctx context.Context, scopes ...string) *Client {
 	return client
 }
 
-func TestIntegration_BucketMethods(t *testing.T) {
+func TestIntegration_BucketCreateDelete(t *testing.T) {
 	ctx := context.Background()
 	client := testConfig(ctx, t)
 	defer client.Close()
-	h := testHelper{t}
 
 	projectID := testutil.ProjID()
-	newBucketName := uidSpace.New()
-	b := client.Bucket(newBucketName)
-	// Test Create and Delete.
-	h.mustCreate(b, projectID, nil)
-	attrs := h.mustBucketAttrs(b)
-	if got, want := attrs.MetaGeneration, int64(1); got != want {
-		t.Errorf("got metagen %d, want %d", got, want)
-	}
-	if got, want := attrs.StorageClass, "STANDARD"; got != want {
-		t.Errorf("got storage class %q, want %q", got, want)
-	}
-	if attrs.VersioningEnabled {
-		t.Error("got versioning enabled, wanted it disabled")
-	}
-	if attrs.LocationType == "" {
-		t.Error("got an empty LocationType")
-	}
-	if attrs.ProjectNumber == 0 {
-		t.Errorf("got a zero ProjectNumber")
-	}
-	h.mustDeleteBucket(b)
 
-	// Test Create and Delete with attributes.
 	labels := map[string]string{
 		"l1":    "v1",
 		"empty": "",
 	}
-	attrs = &BucketAttrs{
-		StorageClass:      "NEARLINE",
-		VersioningEnabled: true,
-		Labels:            labels,
-		Lifecycle: Lifecycle{
```

Comment:
Was setting the lifecycle not validated previously? That's not great.

Reply:
It was not :/